src/accumulation/point_cloud_accumulator.py

```diff
@@ -64,15 +68,19 @@ def merge(self,
                                                                 frame_point_cloud=frame_point_cloud)
 
             current_point_cloud = accumulation_strategy.on_merge(initial_point_cloud=current_point_cloud,
-                                                                 next_point_cloud=next_point_cloud)
+                                                                 next_point_cloud=next_point_cloud,
+                                                                 frame_no=i)
 
         return current_point_cloud
 
     def __load_instance_point_cloud(self,
                                     frame_id: str,
                                     instance_id: str,
                                     frame_point_cloud: LidarPointCloud) -> np.ndarray[float]:
-        return get_instance_point_cloud(frame_id=frame_id,
+
+        instance_ptcl = get_instance_point_cloud(frame_id=frame_id,
```

Comment:
what was the problem with inline?

Reply:
not a problem, i just needed it there so i could export the loaded instance and see what it looks like :) there was export script below. I could fix it if necessary